### ISSUE TITLE
Bokeh show_app is currently unsupported

### ISSUE THREAD

**Issue by uomodellamansarda** [googlecolab/colabtools#1800]
Bug report for Colab: http://colab.research.google.com/.

- Describe the current behavior:
Bokeh library is not working anymore on Colab, but is working on Jupyter notebook
- Describe the expected behavior:
Last year until March 2020 it was working
- The web browser you are using (Chrome, Firefox, Safari, etc.):
Chrome
- Link (not screenshot!) to a *minimal, public, self-contained* notebook that
  reproduces this issue (click the *Share* button, then *Get Shareable Link*):
![bokeh](https://user-images.githubusercontent.com/18501763/103464971-43723e80-4d38-11eb-9bc5-bb634ffb2171.png)

**Comment by blois** [comment]
Do you have a full repro for this available?

**Comment by uomodellamansarda** [comment]
Yep, I will create and example and share here the link to the full colab :)

**Comment by joamatab** [comment]
here is a colab example not working

https://colab.research.google.com/drive/1W96Re-rINGbQJ8KAhGTD1_fyP5nHpdW7?usp=sharing

**Comment by cperry-goog** [comment]
thanks for the notebook. tracking internally at b/303660166

**Comment by joamatab** [comment]
Thank you for taking a look at this

**Comment by foobarchamp** [comment]
knock knock
Any work update on this?
